bin/healthcheck.py

```diff
@@ -12,5 +12,9 @@
 retries = Retry(total=5, backoff_factor=0.1, status_forcelist=[500, 502, 503, 504])
 session.mount("http://", HTTPAdapter(max_retries=retries))
 if __name__ == "__main__":
-    response = session.get(f"http://0.0.0.0:{PORT}/")
+    url = f"http://0.0.0.0:{PORT}"
+    response = session.get(f"{url}/")
     response.raise_for_status()
+
+    hb_response = session.get(f"{url}/__heartbeat__")
```

Comment:
this shouldn't be retried, because we don't `raise_for_status()`. 

Two options:
- revert 86a3c3b8aa0816
- rewrite this healthcheck using `@backoff` so that it retries only when exceptions are raised